src/DynamoCore/Configuration/IPreferences.cs

```diff
@@ -48,7 +48,7 @@ public interface IPreferences
         /// <summary>
         /// Indicates whether background preview is active or not.
         /// </summary>
-        [Obsolete("Property will be deprecated in Dynamo 3.0, please use BackgroundPreviews")]
+        [Obsolete("Property will be deprecated in a future version of Dynamo, please use BackgroundPreviews")]
```

Comment:
internal

Reply:
What is the plan with all these extended interfaces?

Reply:
I think the plan was to switch to a data type that is more tolerant of changes. Something like a struct or abstract class. I am not sure if we are going to do it for 3.0
@sm6srw @mjkkirschner ?

Reply:
I think it currently is out of scope for 3.0.

Reply:
for 3.0 does it make sense to fold them all back into IPreferences instead of making them internal?